src/main/java/cms/gov/madie/measure/utils/ControllerUtil.java

```diff
@@ -1,5 +1,24 @@
 package cms.gov.madie.measure.utils;
 
+import cms.gov.madie.measure.exceptions.UnauthorizedException;
+import gov.cms.madie.models.access.RoleEnum;
+import gov.cms.madie.models.measure.Measure;
+import org.springframework.util.CollectionUtils;
+
 public class ControllerUtil {
   public static final String TEST_CASES = "/measures/{measureId}/test-cases";
+
+  /** returns true, if the measure is owned by the user or if the measure is shared with the user */
+  public static void verifyAuthorization(String username, Measure measure) {
```

Comment:
Comment and method signature are out of sync. Either return boolean or update comment, please.